samoa-instances/src/main/java/org/apache/samoa/instances/Instances.java

```diff
@@ -69,12 +73,24 @@ public Instances() {
   }
 
   public Instances(Reader reader, int size, int classAttribute) {
-    this.classAttribute = classAttribute;
-    arff = new ArffLoader(reader, 0, classAttribute);
-    this.instanceInformation = arff.getStructure();
-    this.instances = new ArrayList<>();
+	  this.classAttribute = classAttribute;
```

Comment:
There are some minor formatting quirks. Could you please use the standard formatting style file included in SAMOA?